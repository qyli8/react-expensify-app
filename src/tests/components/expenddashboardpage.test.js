import React from 'react';
import {shallow} from 'enzyme'
import ExpendDashBoardPage from '../../components/DashboardPage'
test('should return expenses dashborad page', ()=>{
  const wrapper = shallow(<ExpendDashBoardPage />);
  expect(wrapper).toMatchSnapshot()
})