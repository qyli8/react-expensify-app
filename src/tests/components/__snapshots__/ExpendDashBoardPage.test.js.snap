// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return expenses dashborad page 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
